docs/modules/migration/partials/migration-guide-24.2.adoc

```diff
@@ -209,3 +209,9 @@ Normally, it is not necessary to set this value.
 * No migration is necessary if the `format()` function is called with a single `Date` argument.
 * If the second argument was set to `false`, it can simply be removed (that was the default, anyway).
 * If the second argument was set to `true`, replace it with an options object with the `analyzeInfo` property set.
+
+== Slider (Scout JS)
+
+The `Slider` is now rendered using multiple `<div>` instead of using the browser native `<input type="range">`. It's appearance may change and custom CSS might not work anymore.
```

Comment:
```suggestion
The DOM structure of the `Slider` widget has changed.
Check your custom CSS styles and adjust them if necessary.
```

(Don't focus on what might not work anymore, but tell the reader what to do to fix it.)